10_rest/runnerdave/pkg/puppy/rest.go

```diff
@@ -0,0 +1,98 @@
+package puppy
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	chi "github.com/go-chi/chi"
+)
+
+// RestStorer store for puppies
+type RestStorer struct {
+	Db Storer
+}
+
+const (
+	getPuppyRoute    = "/api/puppy/{id}"
+	postPuppyRoute   = "/api/puppy/"
+	putPuppyRoute    = "/api/puppy/{id}"
+	deletePuppyRoute = "/api/puppy/{id}"
+)
+
+// GetPuppyRoute Route for the GET method
+func (rs *RestStorer) GetPuppyRoute() string {
+	return getPuppyRoute
```

Comment:
why consts and methods? const alone should suffice?

Reply:
good point, it was my clumsy way of exporting them, fixed